pages/instructorCourseAdminSharing/instructorCourseAdminSharing.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express');
+const router = express.Router();
+
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const { v4: uuidv4 } = require('uuid');
+const ERR = require('async-stacktrace');
+const error = require('../../prairielib/lib/error');
+const { InstructorSharing } = require('./instructorCourseAdminSharing.html');
+const sqldb = require('../../prairielib/lib/sql-db');
+const sqlLoader = require('../../prairielib/lib/sql-loader');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+router.get('/', function (req, res, next) {
+  debug('GET /');
+
+  sqldb.queryOneRow(
+    sql.get_course_sharing_info,
```

Comment:
When I visited this page locally for XC 101, "Sharing ID" was NULL. We should either default that column to a random UUID, or populate it on page load if it isn't already set.

Reply:
Ok, sounds good.